plugins/woocommerce-blocks/assets/js/data/cart/default-state.ts

```diff
@@ -29,10 +29,12 @@ import {
 
 const EMPTY_PENDING_QUANTITY: [] = [];
 const EMPTY_PENDING_DELETE: [] = [];
+const EMPTY_PENDING_ADD: [] = [];
 
 export interface CartState {
 	cartItemsPendingQuantity: string[];
 	cartItemsPendingDelete: string[];
+	productsPendingAdd: number[];
```

Comment:
Cart item keys are strings, but during add to cart, we add the _product id_ which is a number